docs/api/spec/openapi.yaml

```diff
@@ -95,6 +95,129 @@ components:
               $ref: '#/components/schemas/ApiUser'
             type: array
         type: object
+    AuditLog:
```

Comment:
This file is autogenerated as discussed, you need to move this into the proper included files